static/js/containers/ContentPolicyPage.js

```diff
@@ -11,7 +11,9 @@ export default class ContentPolicyPage extends React.Component<*, void> {
   render() {
     return (
       <div className="content">
-        <DocumentTitle title={formatTitle("Content Policy")} />
+        <DocumentTitle
+          title={formatTitle("MicroMasters Discussions Community Guidelines")}
+        />
         <div className="main-content">
           <Card className="content-policy">
             <h1>MicroMasters Discussion Guidelines</h1>
```

Comment:
Should we update this also to `MicroMasters Discussion Community Guidelines`?

Reply:
@pdpinch

Reply:
thanks for catching. Yes please.